.changeset/strong-vans-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@crowdstrike/glide-core': patch
+---
+
+Slot assertions now properly validate elements when passed through multiple components.
```

Comment:
Struggled to articulate this problem concisely, but I think this covers it? Open to suggestions.

Reply:
I think this gets into implementation too much, and people won't know what it means. Personally, I'd reword it to simply focus on the Modal issue that these changes resolved.